lib/backend/syncersv1/updateprocessors/configurationprocessor.go

```diff
@@ -230,6 +231,13 @@ func (c *configUpdateProcessor) processAddOrModified(kvp *model.KVPair) ([]*mode
 				switch vt := value.(type) {
 				case string:
 					value = vt
+				case v1.Duration:
+					switch fieldInfo.Tag.Get("configv1timescale") {
+					case "seconds":
+						value = fmt.Sprintf("%.1f", vt.Seconds())
```

Comment:
I think `.1` might be a bit limiting, how about `.3` so that seconds can express millisecond precision and millis can express microseconds.  That should be more than enough.